scripts/globals/spells/burst_ii.lua

```diff
@@ -26,6 +26,8 @@ function onSpellCast(caster, target, spell)
     spellParams.M50 = 2;
     spellParams.M100 = 2;
     spellParams.M200 = 2;
+    spellParams.AMIIburstBonus = (caster:getMerit(MERIT_BURST_II) - 1) * 0.03;
+    spellParams.AMIIaccBonus = (caster:getMerit(MERIT_BURST_II) - 1) * 5;
```

Comment:
Shouldn't these ensure that the bonuses don't become negative if the caster has no merits? ie. mobs casting AMIIs.

Reply:
I vaguely remember someone mentioning that mobs by default had max merits, but I don't know if that applies for job-specific ones. I'll add it in though.